src/writer.c

```diff
@@ -554,9 +554,9 @@ write_sep(SerdWriter* writer, const Sep sep)
 
   // Write newline or space before separator if necessary
   if (pre_line) {
-    TRY(st, write_newline(writer));
+	TRY(st, write_newline(writer));
```

Comment:
nit: Lots of mangled whitespace.  Serd is formatted with clang-format, please take advantage of this to avoid mangled formatting in diffs.  Integrations are available for pretty much every editor under the sun, you can also use `ninja clang-format` to fix it after the fact or just run it manually.

Reply:
clang format now run in latest commit, sorry never used clang-format before. It's neat. Snazzy modern tech.